ci/prow/templates/prow_presubmit_gocoverate_job.yaml

```diff
@@ -18,6 +18,7 @@
         - "--artifacts=$(ARTIFACTS)"
         - "--cov-threshold-percentage=[[.Base.GoCoverageThreshold]]"
         - "--github-token=/etc/covbot-token/token"
+        - "--posting-robot=knative-metrics-robot"
```

Comment:
do we need this, if that is the default?

Reply:
In the future if bot name changes, only the template needs to be updated accordingly.

Reply:
I understand the reqmt here. I am suggesting that we dont need to add this flag as that is already the default value in code